docs/MapColonies/auth.md

```diff
@@ -13,17 +13,36 @@ tags:
 
 # Authentication
 
-In order to make requests either as a service or a client, you will need to identify yourself with a token.
-The token, should be used in **every** request to our services and will enable you to access our services in accordance to the permissions that were given to you. <br/>
+In order to make requests either as a service or a client, you will need to identify yourself with a `token`.
+The `token` should be used in **every** request to our services and will allow you to access our services in accordance to the permissions that were given to you.<br/>
+
+**All services** will require an `API Token`, it will be specified in each service description.
 
 :::caution
 The token might have an **expiration date**
 :::
 
-When making any request, you need to add a special **header** or **query parameter** in which you supply the token.
-
 :::info
 **Important:** only **query parameter** `token` is currently supported
 :::
 
-**All services** will require an `API Token`, it will be specified in the service description.
+## Supplying the token
+
+When making any request, you need to add a special **header** or **query parameter** in which you supply the token.
+
+| **How we supply the token** | **Name** |
+| ----------- | ----------- |
+| Query | token |
+| Header | x-api-key |
+
+## Using the token as a query parameter
```

Comment:
Consider add another column for these examples instead of putting them separatly from the table

Reply:
I think it's better that the examples are seperate and have thier own section. Do you think it's too much?